docker/build.sh

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env bash
+
+storepass="$1"
+keypass="$2"
+if [ -z ${3+x} ]; then
+    runDocker=true
+fi
+
+dockerImage="chrgernoe/yacguide"
```

Comment:
```YacGroup/yacguide```

Reply:
Same as above. I will create this organization.

Reply:
See comment for file `CONTRIBUTION.md`.